README.md

```diff
@@ -1,6 +1,17 @@
 # Kafka
+Apache Kafka is a highly scalable, fast, and fault-tolerant distributed streaming platform. 
+
 This repo publishes the docker image and helm chart for [Apache Kafka](https://kafka.apache.org/).
 
+## How do we use kafka?
+Kafka forms the core of the Hypertrace streaming platform and being used extensively across the services. Kafka usage at Hypertrace can be categorised into 3 categories: Plain vanilla Kafka Clients, Kafka streams and Streaming pipeline jobs. We are migrating streaming pipeline jobs also to Kafka streams.
```

Comment:
I don't think this is true except the migration to kafka streams that may or may not happen and if it is going to happen should reference an issue about that.

Currently, Kafka is used by the collector pinot and flink jobs correct? It would be more honest to like the things that use it, ex hypertrace-ingester (flink), all-views-creator (pinot) and the collector (ocensus) container names.

Reply:
@laxmanchekka , can you please create an issue for kafka-stream migration you mentioned yesterday?
@adriancole , will update the description.